Dockerfile

```diff
@@ -1,49 +1,32 @@
-FROM ros:noetic-ros-core
-
-ENV ROS_WS ros_ws
+FROM ros:noetic-ros-base
 
 # Use bash instead of sh
 SHELL ["/bin/bash", "-c"]
 
-# Update Ubuntu Software repository
-RUN apt update && \
-    apt upgrade -y
-
-RUN apt install -y curl \
-    git
-
-RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
-
-RUN apt install -y \
-    python3-dev \
-    python3-pip \
-    python3-rospkg \
-    nodejs
+WORKDIR /ros_ws
 
-# Python 3 dependencies
-RUN pip3 install \
-        rosdep \
-        rospkg
+# Update Ubuntu Software and Install components
+RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash - && \
+    apt update && \
+    apt upgrade -y && \
```

Comment:
Are we upgrading docker images or we are dropping this for future ones?

Reply:
`apt upgrade` doesn't hurt us a lot (especially while comparing to nodejs build), I'd leave it as it is.

Reply:
If you say so, then I'm resolving this thread.